custom_components/adjustable_bed/beds/octo.py

```
@@ -576,3 +580,169 @@ async def _keepalive_loop(self) -> None:
+
+    async def move_head_down(self) -> None:
+        """Move head motor down."""
+        await self.write_command(
+            self.CMD_HEAD_DOWN,
+            repeat_count=25,
+            repeat_delay_ms=200,
+        )
+
+    async def move_head_stop(self) -> None:
+        """Stop head motor (Star2 doesn't have explicit stop - stop sending commands)."""
+        pass
```

Ruff (0.14.11)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)